hydra_agent/redis_core/graphutils_operations.py

```diff
@@ -40,13 +52,13 @@ def get_processing(self, url: str, resource: dict) -> None:
             collection_members = self.graph_utils.read(
                 match=":collection",
                 where="id='{}'".format(redis_collection_id),
-                ret=".members")
+                ret="")
 
             # Checking if it's the first member to be loaded
-            if collection_members is None:
+            if not 'members' in collection_members:
```

Comment:
I think replacing it with `if 'members' not in collection_mambers` would be more readable.

Reply:
I did this for all occurrences in the PR I'm opening soon, the other way is actually not pep8 conformant